dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -984,6 +1025,13 @@
           "items": {
             "$ref": "#/definitions/HealthcareEntityLink"
           }
+        },
+        "EntityComponentInformation": {
```

Comment:
The name of the property should be camelCase

Reply:
yes

Reply:
Why would it be camelCase? All other properties in the swagger are PascalCase (e.g. HealthcareEntityLink, HealthcareEntity, etc...)

Reply:
Definitions of types like HealthcareEntityLink, HealthcareEntity are PascalCase, but names of properties like confidenceScore should be camelCase.

Reply:
Can you check again. The property name `entityComponentKind` in line 1195 is already camelCase

Reply:
I'm talking about camelCase for EntityComponentInformation which is listed as a property of HealthcareEntity:

https://github.com/Azure/azure-rest-api-specs/blob/10e48404583cb7c6d4bd98b1c461902ca36c32c3/dev/cognitiveservices/data-plane/Language/analyzetext.json#L1043-L1050

The definition (PascalCase) is on L1190: https://github.com/Azure/azure-rest-api-specs/blob/10e48404583cb7c6d4bd98b1c461902ca36c32c3/dev/cognitiveservices/data-plane/Language/analyzetext.json#L1190

Reply:
I finally saw it! - Fixed.